proto/cache.proto

```diff
@@ -30,11 +36,96 @@ message CacheStats {
   int64 total_cached_action_exec_usec = 11;
 }
 
+// Request to retrieve detailed per-request cache stats.
+message GetCacheScoreCardRequest {
+  context.RequestContext request_context = 1;
+
+  // The invocation ID for which to look up cache stats.
+  string invocation_id = 2;
+
+  // Pagination token. Not intended for direct client usage.
+  message PageToken {
+    // The start index in the scorecard results list, after sorting and
+    // filtering results. Number of results to fetch is determined by the
+    // server.
+    int64 index = 1;
+  }
```

Comment:
Can we put this in context.proto (or make pagination.proto)?

Reply:
Done